src/emc/usr_intf/axis/scripts/axis.py

```diff
@@ -3340,12 +3350,17 @@ def units(s, d=1.0):
 
 kinsmodule=inifile.find("KINS", "KINEMATICS").lower()
 kins_is_trivkins = False
+trivkinstype = "none"
 if kinsmodule.split()[0] == "trivkins":
     kins_is_trivkins = True
     trivkinscoords = "XYZABCUVW"
     for item in kinsmodule.split():
         if "coordinates=" in item:
             trivkinscoords = item.split("=")[1]
+        if "kinstype=" in item:
+            trivkinstype = item.split("=")[1]
+        else:
+            trivkinstype = "single"
```

Comment:
I think I see what I need to do but I am not sure how to go about this.
Do I close this pull request and start another or can I somehow modify this pull request?